examples/previews.rs

```diff
@@ -1,20 +1,21 @@
 //! Generates previews of digit collections in a markdown-compatible table
 //! format.
+//! Run with `SAVE=SAVE cargo run --example previews` to write to disk.
```

Comment:
This took me an embarrassingly long time to find :D

Reply:
I honestly forgot how to do it when pulling this repo back up 😆